app/components/Form/SubmitBtns.tsx

```diff
@@ -0,0 +1,108 @@
+import Link from 'next/link';
+import styled from 'styled-components';
+import Button from '@button-inc/bcgov-theme/Button';
+import LoadingSpinner from 'components/LoadingSpinner';
+
+const StyledFlex = styled('div')`
+  display: flex;
+  flex-direction: column;
+
+  @media (min-width: 600px) {
+    flex-direction: row;
+  }
+`;
+
+const StyledButton = styled(Button)`
+  margin: 24px 0;
+  white-space: nowrap;
+  min-width: 178px;
+
+  @media (min-width: 600px) {
+    margin: 0 24px;
+  }
+`;
+
+const StyledToast = styled('div')`
+  background-color: ${(props) => props.theme.color.successGreen};
+  border-radius: 4px;
+  padding: 12px 24px;
+  color: #ffffff;
+  font-size: 16px;
+  transition: all ease-in-out 0.2s 0.2s;
+  text-align: center;
+
+  & a {
+    color: #ffffff;
+  }
+`;
+
+type Props = {
+  disabled: boolean;
+  formData: any;
+  isSubmitPage: boolean;
+  isUpdating: boolean;
+  saveAsDraft: boolean;
+  saveForm: any;
+  status: string;
+};
+
+const SubmitBtns = ({
+  disabled,
+  formData,
+  isSubmitPage,
+  isUpdating,
+  saveAsDraft,
```

Comment:
I read this as "should we save as draft", but from what I'm reading it's closer to "has the save as draft completed". Thoughts on changing the name?

Reply:
yeah this one I was stuck on the naming for a bit. Happy to change it but `hasTheSaveAsDraftCompleted` seems a bit verbose. I was just looking at it as saveAsDraft is true, if not then we use saved....not ideal but I'm happy to change it if you have a good name.

Reply:
I'm good with a long name if you are, so long as it's clear! Could also add a js-doc comment around the prop/variable where it's declared but that's not the style of the rest of the app.